packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -45,31 +44,54 @@ const makeDust = brand => AmountMath.make(brand, 1n);
  * @param {ZCFSeat} poolSeat
  * @param {ShareWorth} shareWorth
  * @param {Brand} USDC
+ * @param {Amount<'nat'>} outstandingLends
  */
-const checkPoolBalance = (poolSeat, shareWorth, USDC) => {
+const checkPoolBalance = (poolSeat, shareWorth, USDC, outstandingLends) => {
   const available = poolSeat.getAmountAllocated('USDC', USDC);
   const dust = makeDust(USDC);
-  isEqual(add(available, dust), shareWorth.numerator) ||
+  const virtualTotal = add(add(available, dust), outstandingLends);
+  isEqual(virtualTotal, shareWorth.numerator) ||
     Fail`🚨 pool balance ${q(available)} inconsistent with shareWorth ${q(shareWorth)}`;
 };
 
+/**
+ * @typedef {{
+ *  Principal: Amount<'nat'>;
+ *  PoolFee: Amount<'nat'>;
+ *  ContractFee: Amount<'nat'>;
+ * }} RepayAmountKWR
+ */
+
+/**
+ * @typedef {{
+ *  Principal: Payment<'nat'>;
+ *  PoolFee: Payment<'nat'>;
+ *  ContractFee: Payment<'nat'>;
+ * }} RepayPaymentKWR
+ */
+
 /**
  * @param {Zone} zone
- * @param {ZCF} zcf
- * @param {Brand<'nat'>} USDC
+ * @param {object} powers
+ * @param {ZCF} powers.zcf
+ * @param {{brand: Brand<'nat'>; issuer: Issuer<'nat'>;}} powers.usdc
```

Comment:
I don't see any use of `usdc.issuer`. Why this change?

Reply:
vestigial, removed